src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -130,7 +134,13 @@ enum {
     /* 18 */ ENHORSE_18,
     /* 19 */ ENHORSE_19,
     /* 20 */ ENHORSE_20,
-};
+} EnHorseParam;
+
+/**
+ * `paramtype` should be `ENHORSE_PARAMTYPE_BANDIT`, `ENHORSE_PARAMTYPE_4000` or `ENHORSE_PARAMTYPE_DONKEY`
+ * `lower` should be a value of the enum `EnHorseParam`
+ */
+#define ENHORSE_PARAM(paramtype, lower) ((paramtype) | (lower))
```

Comment:
```suggestion
#define ENHORSE_PARAMS(paramtype, lower) ((paramtype) | (lower))
```
Small nit, but I think its better since the struct member is also plural